packages/svelte/messages/compile-errors/script.md

```diff
@@ -78,6 +78,10 @@
 
 > Cannot use `$props()` more than once
 
+## props_illegal_name
+
+> Declaring or accessing a property starting with `$$` is illegal (they are reserved for Svelte internals)
```

Comment:
Should it be prop, rather than property?

Reply:
Yeah I think that would be better because it more clearly hints that this is the prop-property, not some random property